src/System.Collections/tests/BitArray/BitArray_GetSetTests.cs

```diff
@@ -8,19 +8,29 @@
 
 namespace System.Collections.Tests
 {
-    public class BitArray_GetSetTests
+    public static class BitArray_GetSetTests
     {
-        private static BitArray s_allTrue = new BitArray(320, true);
-        private static BitArray s_allFalse = new BitArray(320, false);
-        private static BitArray s_alternating = new BitArray(Enumerable.Repeat(unchecked((int)0xaaaaaaaa), 10).ToArray());
+        private const int BitsPerByte = BitArray_CtorTests.BitsPerByte;
+        private const int BitsPerInt32 = BitArray_CtorTests.BitsPerInt32;
```

Comment:
Rather than referring to another test class, let's just put the 8 and 32 here.  Same in the similar case below.